fast_neural_style/neural_style/neural_style.py

```diff
@@ -0,0 +1,232 @@
+    content_transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Lambda(lambda x: x.mul(255))
+    ])
+    content_image = content_transform(content_image)
+    content_image = content_image.unsqueeze(0)
+    if args.cuda:
+        content_image = content_image.cuda()
+    content_image = Variable(content_image)
+
+    style_model = TransformerNet()
+    style_model.load_state_dict(torch.load(args.model))
+    if args.cuda:
+        style_model.cuda()
+
+    output = style_model(content_image)
+
+    if args.cuda:
+        output = output.cpu()
+    output_data = output.data[0]
+    utils.save_image(args.output_image, output_data)
+
+
+def main():
+    main_arg_parser = argparse.ArgumentParser(description="parser for fast-neural-style")
+    subparsers = main_arg_parser.add_subparsers(title="subcommands", dest="subcommand")
+
+    train_arg_parser = subparsers.add_parser("train", help="parser for training arguments")
+    train_arg_parser.add_argument("--epochs", type=int, default=2,
+                                  help="number of training epochs, default is 2")
+    train_arg_parser.add_argument("--batch-size", type=int, default=4,
+                                  help="batch size for training, default is 4")
+    train_arg_parser.add_argument("--dataset", type=str, required=True,
+                                  help="path to training dataset, the path should point to a folder "
+                                       "containing another folder with all the training images")
+    train_arg_parser.add_argument("--style-image", type=str, default="images/style-images/mosaic.jpg",
+                                  help="path to style-image")
+    train_arg_parser.add_argument("--save-model-dir", type=str, required=True,
+                                  help="path to folder where trained model will be saved.")
+    train_arg_parser.add_argument("--checkpoint-model-dir", type=str, default=None,
+                                  help="path to folder where checkpoints of trained models will be saved")
+    train_arg_parser.add_argument("--image-size", type=int, default=256,
+                                  help="size of training images, default is 256 X 256")
+    train_arg_parser.add_argument("--style-size", type=int, default=None,
+                                  help="size of style-image, default is the original size of style image")
+    train_arg_parser.add_argument("--cuda", type=int, required=True,
+                                  help="set it to 1 for running on GPU, 0 for CPU")
+    train_arg_parser.add_argument("--seed", type=int, default=42,
+                                  help="random seed for training")
+    train_arg_parser.add_argument("--content-weight", type=float, default=1e5,
+                                  help="weight for content-loss, default is 1e5")
+    train_arg_parser.add_argument("--style-weight", type=float, default=1e10,
```

Comment:
When I was using the pretrained VGG Caffe model with RGB values in the range `[0-255] ` I had the `content_weight` and `style_weight` on the same scale as the one listed in the [original paper](http://cs.stanford.edu/people/jcjohns/eccv16/) and other implementations, but when I started using pytorch's pretrained vgg which had RGB values in the range `[0-1]` I had to increase the scale of the cost lambdas.

I discussed this with a friend who also had to change the scale of cost lambdas when he was using the pretrained VGG for Keras(theano-backend).